custom_components/waste_collection_schedule/waste_collection_schedule/source/edpevent_se.py

```diff
@@ -192,19 +192,38 @@ def fetch(self):
         response = requests.get(getUrl, params=params, timeout=30)
 
         data = json.loads(response.text)
-
+        #print(json.dumps(data, indent=2))
```

Comment:
```suggestion
        #print(json.dumps(data, indent=2))
```
Looks like a debug left over.